src/Components/Reviews/ReviewList.jsx

```diff
@@ -1,15 +1,15 @@
 import React, { useState } from 'react';
-import ReviewListEntry from './ReviewListEntry.jsx'
+import ReviewListEntry from './ReviewListEntry.jsx';
 
-const ReviewList = ({reviews, setReviews, formatDate}) => {
+const ReviewList = ({reviews, setReviews }) => {
 
   return (
     <div className='reviewList'>
       {reviews.results.map((review) =>
-        <ReviewListEntry review={review} setReviews={setReviews} formatDate={formatDate} key={review.review_id}/>
+        <ReviewListEntry review={review} setReviews={setReviews} key={review.review_id || null}/>
```

Comment:
Not sure the reasoning on `key={review.review_id || null}` here. Is this to deal with when there are no values in state? Because if so it should be handled before mapping. The key shouldn't be set to null and should always have a value so I think this shouldn't ever. Not sure when it would be returning null because if the review exists, the review_id should exist.